### PR TITLE
Feature/full directors register test

### PR DESCRIPTION
**Pull request by danfalc**

### CODE REVIEW

File: cypress/integration/location_of_company_records/change_location_of_directors_register.spec.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+const manageRegistersPreFilingPage = new ManageRegistersPreFilingPage();
+const submissionConfirmationPage = new SubmissionConfirmationPage();
+const changeDirectorsRegisterLocationPage = new ChangeDirectorsRegisterLocationPage();
+const basePage = new BasePage();
+
+describe('Elect to Hold Directors Register on the Public Register', () => {
+    it('EH01 check all screens prior to submission', () => {
+
+        // Select form from overview
+        companyOverview.selectAllForms();
```
Comment (rsuller) [inline cypress/integration/location_of_company_records/change_location_of_directors_register.spec.js]: You can limit the code duplication here. From lines 19 - 23 if you move the code to a `beforeEach` block inside the `describe` section, that code will be executed before each `it` test.

File: cypress/support/page_objects/ChangeDirectorsRegisterLocationPage.js [hunk 2]
```diff
@@ -1,20 +1,74 @@
...
-        cy.accessibilityCheck();
+        } else {
+            cy.get('#elect-to-public').click();
 
-    }
+            //Check after clicking the radio button
+            cy.accessibilityCheck();
 
+            //Confirm decision by clicking the checkbox
+            cy.get('input[type="checkbox"][name="changeRegisterLocationDirectors.registerLocationControl.confirm_ckbox"]').click();
```
Comment (rsuller) [inline cypress/support/page_objects/ChangeDirectorsRegisterLocationPage.js]: Wild `css` selector. We could try `name$='confirm_ckbox'` but in the grand scheme of things not essential.
Also worth formatting this file `Alt+Shift+F`

File: cypress/integration/location_of_company_records/change_location_of_directors_register.spec.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        allFormsPage.selectChangeRoAndCompanyRecords()
+        .selectEh01();
```
Comment (rsuller) [inline cypress/integration/location_of_company_records/change_location_of_directors_register.spec.js]: Formatting of this appears to prefer indenting chained methods. `Alt+Shift+F`

File: cypress/support/page_objects/ChangeDirectorsRegisterLocationPage.js (hunk 2 above)
Comment (danfalc) [reply cypress/support/page_objects/ChangeDirectorsRegisterLocationPage.js]: Agreed

File: cypress/integration/location_of_company_records/change_location_of_directors_register.spec.js (hunk 1 above)
Comment (danfalc) [reply cypress/integration/location_of_company_records/change_location_of_directors_register.spec.js]: Ok I'll sort it

File: cypress/integration/location_of_company_records/change_location_of_directors_register.spec.js (hunk 3 above)
Comment (danfalc) [reply cypress/integration/location_of_company_records/change_location_of_directors_register.spec.js]: Sorted

**Pull request merged**